repos/fsharp.proj

```diff
@@ -1,37 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
-  <PropertyGroup>
-    <PackagesOutput>$(ProjectDirectory)/BuildFromSource/Release/bin</PackagesOutput>
-    <BuildRevision>181017</BuildRevision>
-    <BuildCommand>$(ProjectDirectory)/src/buildfromsource$(ShellExtension)</BuildCommand>
-    <RepoApiImplemented>false</RepoApiImplemented>
-    <!-- This repo is not currently built in ProdCon -->
-    <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
-    <OfficialBuildId>20190131.4</OfficialBuildId>
-  </PropertyGroup>
-
-  <!-- Auto-detect the root NuGet.Config, but also update the build-from-source-specific one. -->
-  <ItemGroup>
-    <NuGetConfigFiles Include="$(ProjectDirectory)/src/buildfromsource/NuGet.config" />
-  </ItemGroup>
 
   <PropertyGroup>
```

Comment:
Do you need to set the OfficialBuildId property here to get the right version generated?

Reply:
Is there a way to either auto-generate the version, or auto-infer it?  Once the regular updates to the submodule are incorporated, won't that need to be updated, too?  Otherwise we'd be statically setting a build version, but auto-updating the source which would make the build version essentially meaningless.

Reply:
(Also, if we do need to set this version, what do we set it to?  The regular daily build that happens to correspond do the sources in the submodule?)

Reply:
Yes.  This version should get updated when we get an auto-update.  @crummel can confirm since he's setting that up.

Reply:
Thanks, I'll add `<OfficialBuildId>20190411.1</OfficialBuildId>` which is the build number that corresponds to the sources.